testing/spec/FormulaSpec.js

```diff
@@ -158,6 +158,8 @@ describe("compound creation", function(){
 
         });
 
+        //TODO: make more tests here. need to test more complex compounds
+
```

Comment:
Currently only have tests for very simple compounds. Should create more complex tests.